src/datastore/postgres-store.ts

```diff
@@ -685,18 +692,22 @@ export class PgDataStore
   extends (EventEmitter as { new (): DataStoreEventEmitter })
   implements DataStore {
   readonly pool: Pool;
+  readonly primaryPool: Pool;
```

Comment:
Should there be new code added to the `close()` function to destroy this pool instance?